automation/Extension/firefox/lib/loggingdb.js

```diff
@@ -7,18 +7,27 @@ var visitID = null;
 var debugging = false;
 var sqliteAggregator = null;
 var ldbAggregator = null;
+var logAggregator = null;
 var listeningSocket = null;
 
-exports.open = function(sqliteAddress, ldbAddress, crawlID) {
-    if (sqliteAddress == null && ldbAddress == null && crawlID == '') {
+exports.open = function(sqliteAddress, ldbAddress, logAddress, curr_crawlID) {
```

Comment:
There was a javascript variable scope bug that didn't crop up before because crawlID was never actually used here. Basically, the naming collision between the function scoped "crawlID" and the global "crawlID" meant that the global crawlID was never getting set.